tests/osaka/eip7823_modexp_upper_bounds/test_modexp_upper_bounds.py

```diff
@@ -270,6 +270,36 @@ def test_modexp_upper_bounds(
     state_test(pre=pre, tx=tx, post=post)
 
 
+@pytest.mark.valid_from("Prague")
+@pytest.mark.parametrize(
+    "modexp_input,modexp_expected,call_succeeds",
+    [
+        pytest.param(
+            ModExpInput(
+                base=b"",
+                exponent=b"",
+                modulus=b"",
+                declared_exponent_length=2**32,
+                declared_modulus_length=1,
+            ),
+            Spec.modexp_error,
+            False,
+        ),
+    ],
+)
+def test_modexp_upper_bounds_pre_osaka(
```

Comment:
This test case is related to code changes introduced for EIP-7823 but we need to run it _before_ Osaka. There are no such tests for EIP-7823. Maybe we should update the existing `test_modexp_upper_bounds` to run from Prague, but some test cases may have different call result.

Reply:
Created an issue for this, thanks!